[2021-02-18 22:42:05,184] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:41:00+00:00 [queued]>
[2021-02-18 22:42:05,190] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:41:00+00:00 [queued]>
[2021-02-18 22:42:05,190] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-02-18 22:42:05,190] {taskinstance.py:1018} INFO - Starting attempt 1 of 2
[2021-02-18 22:42:05,190] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-02-18 22:42:05,201] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): get_tweets_by_user_list> on 2021-02-18T22:41:00+00:00
[2021-02-18 22:42:05,203] {standard_task_runner.py:51} INFO - Started process 18575 to run task
[2021-02-18 22:42:05,208] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'tweets_by_user', 'get_tweets_by_user_list', '2021-02-18T22:41:00+00:00', '--job-id', '1138', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/tweets_by_user.py', '--cfg-path', '/tmp/tmppd84016f']
[2021-02-18 22:42:05,211] {standard_task_runner.py:76} INFO - Job 1138: Subtask get_tweets_by_user_list
[2021-02-18 22:42:05,240] {logging_mixin.py:103} INFO - Running <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:41:00+00:00 [running]> on host ip-172-31-3-59.ec2.internal
[2021-02-18 22:42:05,267] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=tweets_by_user
AIRFLOW_CTX_TASK_ID=get_tweets_by_user_list
AIRFLOW_CTX_EXECUTION_DATE=2021-02-18T22:41:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-18T22:41:00+00:00
[2021-02-18 22:42:05,268] {logging_mixin.py:103} INFO - Autenticação no twitter
[2021-02-18 22:42:05,283] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:42:05,476] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:42:06,034] {logging_mixin.py:103} INFO - 200
[2021-02-18 22:42:06,044] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:42:06,202] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:42:06,605] {logging_mixin.py:103} INFO - 400
[2021-02-18 22:42:06,613] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:42:06,781] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:42:07,127] {logging_mixin.py:103} INFO - 600
[2021-02-18 22:42:07,130] {logging_mixin.py:103} INFO - 0:00:01.862892
[2021-02-18 22:42:07,130] {logging_mixin.py:103} INFO - 600
[2021-02-18 22:42:09,052] {logging_mixin.py:103} INFO - 0:00:01.921868
[2021-02-18 22:42:09,096] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:42:09,119] {logging_mixin.py:103} INFO - 0:00:00.066486
[2021-02-18 22:42:09,119] {logging_mixin.py:103} INFO - 600 devidamente inseridas no banco
[2021-02-18 22:42:09,121] {python.py:118} INFO - Done. Returned value was: None
[2021-02-18 22:42:09,124] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=tweets_by_user, task_id=get_tweets_by_user_list, execution_date=20210218T224100, start_date=20210218T224205, end_date=20210218T224209
[2021-02-18 22:42:09,145] {taskinstance.py:1195} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-02-18 22:42:09,152] {local_task_job.py:118} INFO - Task exited with return code 0
